tools/ci/tc/decision.py

```diff
@@ -0,0 +1,314 @@
+import argparse
+import json
+import logging
+import os
+import re
+import subprocess
+from collections import OrderedDict
+
+import taskcluster
+from six import iteritems, itervalues
+
+from . import taskgraph
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+logging.basicConfig()
+logger = logging.getLogger()
+
+
+def get_triggers(event):
+    # Set some variables that we use to get the commits on the current branch
+    ref_prefix = "refs/heads/"
+    pull_request = "pull_request" in event
```

Comment:
```suggestion
    is_pr = "pull_request" in event
```

To match how it's later used, and it would be clearer here as well.